frontend/src/components/TimeTravel.tsx

```diff
@@ -104,9 +103,8 @@ export const TimeTravel = ({
         startDate={currentWeekArray[0]}
         endDate={currentWeekArray[4]}
         selectsRange
-        locale={sv}
```

Comment:
but not here?

Reply:
Right. We need to make a decision about this then.

Do we want to have Swedish language and weeks in the calendar or something else?

Reply:
* The week numbers should be based on the Swedish standard.
* The week starts on Monday.
* All text should be in english.

Reply:
In other words, it's ok to keep it as it is now, swedish in the calendar and have the correct week numbering scheme. Lets just work on one issue at a time, and if you find other things that we might need to work on during development create issues for this and raise it with the me and Jonas so we can prioritize it.